tests/page-objects/contacts/admin-user.wdio.page.js

```diff
@@ -0,0 +1,68 @@
+const addUserButton = () => $('a#add-user');
+const addUserDialog = () => $('h2=Add user');
+const userName = () => $('#edit-username');
+const userFullName = () => $('#fullname');
+const userRole = () => $('#role');
+const userPlace = () => $('//span[@aria-labelledby="select2-facilitySelect-container"]');
+const userAssociatedContact = () => $('//span[@aria-labelledby="select2-contactSelect-container"]');
+const userPassword = () => $('#edit-password');
+const userConfirmPassword = () => $('#edit-password-confirm');
+const saveUserButton = () => $('a=Add user');
```

Comment:
Have you try using the button classes here? I worry about using text values. 

`const saveUserButton = () => $('.btn.submit');`

Reply:
I have added the `id` attribute to the button, let me know if it doesn't make sense